Orange/canvas/report/report.py

```diff
@@ -162,8 +163,7 @@ def report_plot(self, name=None, plot=None):
             self.report_html += get_html_img(plot.scene())
 
     # noinspection PyBroadException
-    def report_table(self, name, table=None, header_rows=0, header_columns=0,
-                     num_format=None):
+    def report_table(self, name, table=None, header_rows=0, header_columns=0, confusion_rows=0, num_format=None):
```

Comment:
I still don't feel this is the right approach. First, you name the parameter `confusion_rows`, which possibly makes sense only when working with confusion matrices; but `report_table()` is meant to report _any_ kind of table. Second, by adding your new parameter somewhere in the middle of the parameters list (as opposed to the end), you break the function's existing API and should, at the very least, ensure no other calls rely on the old parameter order. Finally, even if you think you absolutely must use the number of rows in the table, you can get them from the `table` parameter, which is an instance of table model [or something compatible](https://github.com/biolab/orange3/pull/1097/files#diff-a8517f05dab824c1e209188aaf6661d5L243) (in either case, it knows for certain how many rows it has). So no, I don't think this new parameter is ok.